test/index.ts

```diff
@@ -41,8 +46,17 @@ export function runTests(tests: Test, cwd: string = process.cwd()): void {
         });
     }
 
-    if (shouldRunBenchmarks) {
-        describe(`benchmarks`, () => {});
+    if (shouldRunBenchmarks && canisterNames !== undefined) {
+        const canisterNamesArray = Array.isArray(canisterNames)
+            ? canisterNames
+            : [canisterNames];
+
+        canisterNamesArray.forEach((canisterName) => {
+            describe(`benchmarks for ${canisterName}`, () => {
```

Comment:
I'm thinking this should just be one global benchmarks call like the typechecks, it just does the benchmark stuff. All of the stats are collected and put into one markdown file and one json file

Reply:
done